dio/lib/dio.dart

```diff
@@ -1,6 +1,6 @@
-/// A powerful Http client for Dart, which supports Interceptors,
-/// Global configuration, FormData, File downloading etc. and Dio is
-/// very easy to use.
+/// A powerful HTTP package for Dart/Flutter, which supports Global settings,
```

Comment:
I would keep it as HTTP client

```suggestion
/// A powerful HTTP client for Dart and Flutter, which supports global settings,
```